src/app/test/test.component.ts

```diff
@@ -16,7 +16,7 @@ import { NodeLinkStrategy } from '../shared/enums/compare-method';
 import { TestFolderTreeComponent } from './test-folder-tree/test-folder-tree.component';
 import { ToastComponent } from '../shared/components/toast/toast.component';
 import { NgIf, NgFor } from '@angular/common';
-import { ReactiveFormsModule, FormsModule, NgModel } from '@angular/forms';
+import { ReactiveFormsModule, FormsModule, NgModel, FormControl } from '@angular/forms';
 import { ButtonComponent } from '../shared/components/button/button.component';
 
```

Comment:
```suggestion
export const updatePathActionConst = ['move', 'copy'] as const;
export type UpdatePathAction = (typeof updatePathActionConst)[number]
```

Reply:
I added the change. Should the type not be shown behind line 21 in the example for the sake of following conventions? I get an error when I try that however.

Reply:
No it's fine like this, check the [conventions](https://github.com/frankframework/frontend-conventions/blob/master/typescript.md#enums)